packages/arb-token-bridge-ui/src/components/TransferPanel/Routes/ArbitrumRoute.tsx

```diff
@@ -0,0 +1,164 @@
+import { useNativeCurrency } from '../../../hooks/useNativeCurrency'
+import { ether } from '../../../constants'
+import { CommonAddress } from '../../../util/CommonAddressUtils'
+import {
+  getOrbitDepositDuration,
+  getStandardDepositDuration,
+  getWithdrawalDuration
+} from '../../../hooks/useTransferDuration'
+import { isNetwork } from '../../../util/networks'
+import dayjs from 'dayjs'
+import { useSelectedToken } from '../../../hooks/useSelectedToken'
+import { isTokenNativeUSDC } from '../../../util/TokenUtils'
+import { useRouteStore } from '../hooks/useRouteStore'
+
+const commonUsdcToken = {
+  decimals: 6,
+  address: CommonAddress.Ethereum.USDC
+}
+
+const bridgedUsdcToken = {
+  ...commonUsdcToken,
+  name: 'Bridged USDC',
+  symbol: 'USDC.e',
+  l2Address: CommonAddress.ArbitrumOne['USDC.e']
+}
+
+const nativeUsdcToken = {
+  ...commonUsdcToken,
+  name: 'USDC',
+  symbol: 'USDC',
+  l2Address: CommonAddress.ArbitrumOne.USDC
+}
+
+function getDuration({
+  isTestnet,
+  sourceChainId,
+  isTeleportMode,
+  isWithdrawal,
+  isOrbitChain
+}: {
+  isTestnet: boolean
+  sourceChainId: number
+  isTeleportMode: boolean
+  isWithdrawal: boolean
+  isOrbitChain: boolean
+}) {
+  if (isTeleportMode) {
+    return (
+      getStandardDepositDuration(isTestnet) + getOrbitDepositDuration(isTestnet)
+    )
+  }
+
+  if (isWithdrawal) {
+    return getWithdrawalDuration({
+      createdAt: dayjs().valueOf(),
+      sourceChainId: sourceChainId
+    })
+  }
+
+  if (isOrbitChain) {
+    return getOrbitDepositDuration(isTestnet)
+  }
+
+  return getStandardDepositDuration(isTestnet)
+}
+
+export function ArbitrumRoute({
+  onRouteSelected
+}: Pick<RouteProps, 'onRouteSelected'>) {
+  const amount = useAmountBigNumber()
+  const [networks] = useNetworks()
+  const {
+    childChain,
+    childChainProvider,
+    parentChainProvider,
+    isDepositMode,
+    isTeleportMode
+  } = useNetworksRelationship(networks)
+  const [token] = useSelectedToken()
+  const { isTestnet, isOrbitChain } = isNetwork(childChain.id)
+  const {
+    status: gasSummaryStatus,
+    estimatedParentChainGasFees,
+    estimatedChildChainGasFees
+  } = useGasSummary()
+  const childChainNativeCurrency = useNativeCurrency({
+    provider: childChainProvider
+  })
+  const parentChainNativeCurrency = useNativeCurrency({
+    provider: parentChainProvider
+  })
+  const { selectedRoute } = useRouteStore()
+
+  const estimatedTotalGasFees =
+    gasSummaryStatus === 'loading' ||
+    typeof estimatedChildChainGasFees == 'undefined' ||
+    typeof estimatedParentChainGasFees == 'undefined'
+      ? undefined
+      : estimatedParentChainGasFees + estimatedChildChainGasFees
+
+  /**
+   * If source and destination chains are using the same currency, we display combined cost for both child and parent chain.
+   * If they use a different currency, we display cost for child chain only
+   */
+  const gasCost =
```

Comment:
Might need to revisit the gas logic here because I get `N/A` being displayed intermittently.
(This was while transferring 0.00001 XAI  from ArbOne to XAI chain, but I saw this issue in other cases as well)

<img width="400" alt="image" src="https://github.com/user-attachments/assets/9df20f9c-f6c7-4b83-9611-1b4fe98a944b" />

<hr />

Original 
<img width="200" alt="image" src="https://github.com/user-attachments/assets/0ef5d349-ce07-436a-963f-b44af8b314c2" />